packages/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -30,12 +32,18 @@ export interface TransactionObject {
 	readonly type: number;
 }
 
+export interface SignatureObject {
+	transactionId: string;
+	signature: string;
+}
 export interface TransactionFunctions {
 	containsUniqueData(): boolean;
 	isExpired(date: Date): boolean;
 	verifyAgainstOtherTransactions(
 		otherTransactions: ReadonlyArray<Transaction>,
 	): boolean;
+	addSignature(signature: SignatureObject): boolean;
```

Comment:
`addSignature` takes `signature: string` and `publicKey:string` separately instead of object.